lib/cbus_elixir/app/meetings.ex

```diff
@@ -7,6 +7,51 @@ defmodule CbusElixir.App.Meetings do
 
   alias CbusElixir.Repo
   alias CbusElixir.App.Meeting
+  alias CbusElixir.Pagination
+  alias CbusElixir.App.Speaker
+  alias CbusElixir.App.Attendee
+
+  @doc """
+  Fetch a paged result for meetings
+  """
+  def meetings_for_page(page \\ 1, per_page \\ 5)
+
+  @spec meetings_for_page(integer(), integer()) :: map()
+  def meetings_for_page(page, per_page) do
+    query =
+      from(m in Meeting,
+        join: s in assoc(m, :speakers),
+        where: m.date < ^DateTime.utc_now(),
+        order_by: [desc: m.date],
+        preload: [speakers: s]
+      )
+
+      Pagination.paginate(query, page, per_page)
+  end
+
+
+  def attendees_for_meeting(id) do
+    query = 
+      from(a in Attendee,
+        where: a.meeting_id == ^id,
+        select: a.name
+      )
+
+    query
+    |> Repo.all()
+  end
+
+  def attendees_for_meetings(id) do
```

Comment:
This function name is confusing. It seems to be returning Meetings, but the function name implies that the return type would be `Attendee.t()`

Reply:
It is returning attendees, isn't it? Selects the name where the id passed is matched to the meeting_id on attendees.